Automattic/wp-calypso
# Stats: Break up monolithic _stats.scss file into appropriate component files

As part of the stats maintenance effort underway, we need to take the `assets/stylesheets/sections/_stats.scss` file and break it down:

- It needs to be moved into the `client/my-sites/stats/` directory
- The rules inside of it need to be shuffled into their various component-specific directories, which now exist.

This PR touches the following components:
- ~~StatsChartTabs~~
- StatsComments
- StatsOverviewPlaceholder

In order to test:
- ~~Verify that the tabs show up correctly on the Day/Week/Month/Year charts~~
- Verify that the comments list displays and functions correctly on the Insights page
- Verify that when localStorage is cleared, you get visual placeholders on the modules while they're loading from the API
